api/client/samples/analysis_kits/stocks_to_use_price_model/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pandas==0.24.2
+matplotlib==3.1.1
+scikit-learn==0.21.2
```

Comment:
Let's add `jupyter` here and make the versions `~=` instead of `==`

Reply:
done